kasa/smart/smartdevice.py

```diff
@@ -78,6 +80,8 @@ async def _initialize_children(self):
     @property
     def children(self) -> Sequence[SmartDevice]:
         """Return list of children."""
+        if self.device_type == DeviceType.WallSwitch:
```

Comment:
Add a short note for this special behavior.

Reply:
Done